core/frontend/src/store/system-information.ts

```diff
@@ -237,6 +254,7 @@ const system_information: SystemInformationStore = getModule(SystemInformationSt
 
 system_information.fetchSystem()
 callPeriodically(system_information.fetchPlatform, 5000)
+callPeriodically(system_information.fetchSerial, 5000)
```

Comment:
not a big fan of these periodic calls... but we need a better solution for these anyway